pkg/controllers/networkinfo/networkinfo_controller.go

```diff
@@ -373,6 +377,27 @@ func (r *NetworkInfoReconciler) Start(mgr ctrl.Manager) error {
 	return nil
 }
 
+func (r *NetworkInfoReconciler) RegisterAllNetworkInfo(ctx context.Context) error {
+	log.Info("Register all NetworkConfigurations")
+	var networkConfigList v1alpha1.VPCNetworkConfigurationList
+	if err := r.Client.List(ctx, &networkConfigList); err != nil {
+		log.Error(err, "Failed to list NetworkInfo")
+		return err
+	}
+	for _, nconfig := range networkConfigList.Items {
+		vname := nconfig.GetName()
+		ninfo, err := buildNetworkConfigInfo(nconfig)
+		if err != nil {
+			log.Error(err, "build network config failed")
```

Comment:
Would you print the failed VPCNetworkConfiguration name in the error log?

Reply:
Done